### ISSUE TITLE
Support RedHat Enterprise & RPM packaging distribution

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#786]
This ticket is about supporting RPM packaging and RPM distribution aside the the debian packaging platform.

Supporting Redhat Enterprise platform with RPM packaging require:
- Analyzing RH's sandboxing capabilities (compared to Ubuntu 12.04)
- Evaluating rewriting requirements of init scripts
- other...?

It must be evaluated and documented how to manage release procedures for RPM.

It must be evaluated and documented how to setup and maintain RPM repositories.

In this context, understanding the possibility to use bintray.com to simplify the process, maybe relevant.
##

**Comment by evilaliv3** [comment]
Deprecated.

**Comment by kituser** [comment]
How far is the "porting to RPM/RedHat" process? Is it possible now, to run GL on RPM based distributions like CentOS and RedHat?

If not, are there any current timeframe set?

**Comment by fpietrosanti** [comment]
@kituser no, if you want to contribute or to sponsor such a deployment, it would be great!

**Comment by MrMEEE** [comment]
@fpietrosanti I would love to help with this.. I have the server starting on a RHEL8, but when trying to connect on either port 80 or 8082 I get a binary response.. 

[root@localhost globaleaks]# curl http://localhost:8082 
Warning: Binary output can mess up your terminal. Use "--output -" to tell 
Warning: curl to output it to your terminal anyway, or consider "--output 
Warning: <FILE>" to save to a file.

Have you got any ideas what might cause this?

**Comment by MrMEEE** [comment]
Forget it, I found the issue..

I now have a running Globaleaks on a RHEL8.. :+1: 

I will do some pull requests to get it in...

**Comment by MrMEEE** [comment]
I plan on doing github->travis->copr integration:
https://gist.github.com/abn/daf262e7e454509df1429c87068923d1

Any comments?

**Comment by evilaliv3** [comment]
This is so cool @MrMEEE! Please go ahead and count on my support for pull review and integration!

Please let me know that i could do to support you in this develolment and i may create an official 'globaleaks' account on copr and grant you development privileges!

Thank you for your support!

Please reach us on https://slack.globaleaks.org so that we could chat about this all.

**Comment by MrMEEE** [comment]
Cool.. I sent you a message on Slack :)

**Comment by yevon** [comment]
Hi! Are there any news regarding this one? We would be very interested in this one in our organization, as they want install it natively in redhat. Any known temporary workarrounds for allowing redhat install? Thanks for your work!
